channels/facebook/utility.js

```diff
@@ -541,8 +542,17 @@ var getCompany = (accessToken) => {
 	})
 
 }
-
+/**
+ * 
+ * For Production installs of Application, 
+ * FB reqires sending the access_token as well as hash of the key/secret
+ * This is ONLY used in production apps
+ * see https://developers.facebook.com/docs/workplace/integrations/third-party
+ */
 var generateProof = (accessToken) => {
+	if (process.env.DEV_INTERGRATION) {
```

Comment:
I'd change the const name to WP_PRODUCTION and check if it's false
Reason being, if I'm not mistaken, on FB messenger we still only need to send just the access token

Reply:
good idea. i like that name better

Reply:
i didn't want to use a flag for this but Facebook sends exactly the same response for custom integrations and 3rd party integrations. 
So there isn't a good way to tell